contents/docs/experiments/no-code-web-experiments.md

```diff
@@ -4,7 +4,7 @@ sidebar: Docs
 showTitle: true
 ---
 
-> 🚧 **NOTE:** No-code web experiments are currently considered in `beta`. To access them, enable the [feature preview](https://us.posthog.com#panel=feature-previews) in your PostHog account. We are keen to gather as much feedback as possible so if you try this out please let us know. You can email [daniel.b@posthog.com](mailto:daniel.@posthog.com), send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow), or use one of our other [support options](/docs/support-options).
+> 🚧 **NOTE:** No-code web experiments are currently considered in `beta`. To access them, enable the [feature preview](https://us.posthog.com#panel=feature-previews) in your PostHog account. You'll also need to define `disable_web_experiments: false` in your Posthog web snippet configuration. We are keen to gather as much feedback as possible so if you try this out please let us know. You can email [daniel.b@posthog.com](mailto:daniel.@posthog.com), send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow), or use one of our other [support options](/docs/support-options).
```

Comment:
```suggestion
> 🚧 **Note:** No-code web experiments are currently considered in `beta`. To access them, enable the [feature preview](https://us.posthog.com#panel=feature-previews) in your PostHog account. You'll also need to define `disable_web_experiments: false` in your Posthog web snippet configuration. 
> We are keen to gather as much feedback as possible so if you try this out please let us know. You can email [daniel.b@posthog.com](mailto:daniel.@posthog.com), send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow), or use one of our other [support options](/docs/support-options).
```

Reply:
@ivanagas Out of curiosity, do you intentionally prefer 1 over 2?

1.

![CleanShot 2024-11-27 at 14 49 22@2x](https://github.com/user-attachments/assets/95dee994-8360-4f68-9ed2-34a44c008e94)

2.

![CleanShot 2024-11-27 at 14 50 07@2x](https://github.com/user-attachments/assets/05c7875b-d21c-4a28-b45f-f5858a4d30b5)

Reply:
Oops, I meant to do

```
>
>
>
```

Reply:
Updated the comment now

Reply:
@ivanagas Perfect, thanks!